forge-gui/res/cardsfolder/upcoming/blade_of_shared_souls.txt

```diff
@@ -0,0 +1,9 @@
+Name:Blade of Shared Souls
+ManaCost:2 U
+Types:Artifact Equipment
+K:For Mirrodin
+T:Mode$ Attached | ValidSource$ Card.Self | TriggerZones$ Battlefield | ValidTarget$ Creature | Execute$ TrigCopy | TriggerDescription$ Whenever CARDNAME becomes attached to a creature, for as long as CARDNAME remains attached to it, you may have that creature become a copy of another target creature you control.
+SVar:TrigCopy:DB$ Clone | ValidTgts$ Creature.YouCtrl+NotTriggeredTarget | TgtPrompt$ Choose another target creature you control | CloneTarget$ Equipped | Duration$ UntilUnattached
+K:Equip:2
+DeckHas:Type$Rebel
```

Comment:
Ability$Token too – maybe even Color$Red ??

Add the Ability$Token to the glider, too